src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -450,13 +452,14 @@ private function writeBuffer(string $message, bool $newLine, int $type): void
 
     private function createBlock(iterable $messages, string $type = null, string $style = null, string $prefix = ' ', bool $padding = false, bool $escape = false): array
     {
+        $this->outputWrapper ??= new OutputWrapper();
```

Comment:
Seems like a micro-optimization. What about creating a new instance every time we call this method?